### PR TITLE
add script for fugacious warnings

### PR DESCRIPTION
**Pull request by wslack**
cc @micahsaul and @konklone - unsure if we need to run hubot locally to test this...

I did verify that hubot doesn't care about https in front of URLs in #transient.

### CODE REVIEW

File: scripts/security.coffee [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#
+# Dependencies: none
+#
+# Configuration: none
+#
+# Commands:
+#   mention "https://fugacious.18f.gov/m/" in any message
+#
+# Notes:
+#   Needs another warning for "fugacio.us/m/"
```
Comment (konklone) [inline scripts/security.coffee]: It looks like it has both warnings?

**Comment by konklone** [comment]
Nice, @wslack! This looks :+1: from me, but someone with more experience with hubot should do the merge.

File: scripts/security.coffee (hunk 1 above)
Comment (wslack) [inline scripts/security.coffee]: oh derp, forgot I went back and added the other expression. will fix now.

**Comment by wslack** [comment]
@shawnbot, @jmcarp, @gemfarmer - can one of y'all check this? Should be quick and easy?

**Comment by wslack** [comment]
Consulted with @jseppi and am now merging!

**Comment by amoose** [comment]
DMs only? that seems a little overkill. sometimes I share things (with 3 max views) to three people in a channel or group. just a thought.

**Comment by konklone** [comment]
Group DMs would also qualify.

**Comment by wslack** [comment]
@amoose I think it's ok since Hubot isn't in any super small channels.

**Pull request merged**